.github/workflows/deployment-e2e.yml

```diff
@@ -39,8 +39,6 @@ jobs:
 
       - name: Install pulumi
         uses: pulumi/setup-pulumi@v2
-        with:
```

Comment:
This will always take the latest pulumi in this case. I'm totally fine with that, it's just not 100% aligned with the rest of the dependencies (since they are pinned)